Project 13/entity/page.ts

```diff
@@ -0,0 +1,42 @@
+import {Shop} from "./shop";
+import {Grid} from "./grid";
+import {Card} from "./card";
+
+export class Page {
+    private shop: Shop;
+    private grid: Grid;
+
+    constructor(shop: Shop, grid: Grid) {
+        this.shop = shop;
+        this.grid = grid;
+    }
+
+    setCards(): void {
+        Shop.allVehicles
+            .map(vehicle => {
```

Comment:
```suggestion
            .map((vehicle: Vehicle) => {
```
предпочитаю более вербозный вариант, поскольку из контекста не всегда понятно о каком типе идет речь, а также это позволит в случае чего при разработке подглянуть в структуру сущности. Опять же - делать так или нет - дело кодстайла и единообразия.